proxy/server.go

```diff
@@ -88,6 +88,7 @@ func NewServer(ctx context.Context, cfg config.Config, manager radio.ManagerServ
 		Handler:      r,
 		ReadTimeout:  time.Second * 10,
 		WriteTimeout: time.Second * 5,
+		BaseContext:  func(l net.Listener) context.Context { return ctx },
```

Comment:
Not sure we need or want this, it's not passing any new values down the chain and the cancellation is already handled by the server itself.